src/js/control-bar/mute-toggle.js

```diff
@@ -65,7 +65,15 @@ class MuteToggle extends Button {
    * @listens click
    */
   handleClick(event) {
-    this.player_.muted(this.player_.muted() ? false : true);
+    const vol = this.player_.volume();
+    const lastVolume = this.player_.lastVolume();
```

Comment:
is it possible for the `lastVolume` to end up being 0? Is it worth handling? At that point, I guess would be to set the volume to 0.5?